nsxt/resource_nsxt_policy_tier0_gateway_interface.go

```diff
@@ -119,20 +121,47 @@ func resourceNsxtPolicyTier0GatewayInterfaceCreate(d *schema.ResourceData, m int
 		return fmt.Errorf("segment_path is mandatory for interface of type %s", ifType)
 	}
 
-	localeService, err := getPolicyTier0GatewayLocaleServiceWithEdgeCluster(tier0ID, connector)
-	if err != nil {
-		return err
-	}
-	if localeService == nil {
-		return fmt.Errorf("Edge cluster is mandatory on gateway %s in order to create interfaces", tier0ID)
+	localeServiceID := ""
+	if isPolicyGlobalManager(m) {
+		localeServices, err := listPolicyTier0GatewayLocaleServices(connector, tier0ID, true)
+		if err != nil {
+			return err
+		}
+		if len(localeServices) == 0 {
+			return fmt.Errorf("Edge cluster is mandatory on gateway %s in order to create interfaces", tier0ID)
+		}
+		for _, objInList := range localeServices {
+			if objInList.EdgeClusterPath != nil {
+				localeServiceID = *objInList.Id
```

Comment:
Here the first locale service that has edge cluster is taken for the interface. But I think we should allow the user to specify the site.

Reply:
Done